src/app/machines/views/MachineList/MachineListControls/MachineActionMenu/MachineActionMenu.tsx

```diff
@@ -74,7 +74,7 @@ const MachineActionMenu = ({
           alwaysShowLifecycle
           className="is-maas-select"
           constrainPanelWidth
-          excludeActions={[NodeActions.IMPORT_IMAGES]}
+          excludeActions={[NodeActions.IMPORT_IMAGES, NodeActions.CHECK_POWER]}
```

Comment:
I'd use this as an opportunity to improve existing code. Clearly, there is some repetition here and props that are common to both components. How about:

```tsx
const commonProps = {
  excludeActions: [NodeActions.IMPORT_IMAGES, NodeActions.CHECK_POWER],
  getTitle,
  hasSelection,
  nodeDisplay: "machine",
  onActionClick: (action: NodeActions) => {
    if (action === NodeActions.TAG && !tagsSeen) {
      setTagsSeen(true);
    }
    const view = Object.values(MachineSidePanelViews).find(
      ([, actionName]) => actionName === action
    );
    if (view) {
      setSidePanelContent({ view });
    }
    sendAnalytics(
      "Machine list action form",
      getNodeActionTitle(action),
      "Open"
    );
  },
};

return (
  <>
    <div className="u-hide--medium u-hide--small">
      <NodeActionMenuGroup
        alwaysShowLifecycle
        {...commonProps}
      />
    </div>
    <div className="u-hide--large">
      <NodeActionMenu
        className="is-maas-select"
        constrainPanelWidth
        menuPosition="left"
        toggleAppearance=""
        toggleClassName="p-action-menu"
        toggleLabel="Menu"
        {...commonProps}
      />
    </div>
  </>
);
```

The above is just a rough example, make sure to check all props.

Reply:
Sounds good, will do :+1: